charts/spire/charts/spire-server/README.md

```diff
@@ -348,6 +348,18 @@ In order to run Tornjak with simple HTTP Connection only, make sure you don't cr
+| `bundlePublisher.awsRolesAnywhereTrustAnchor.enabled`                                        | Enable the AWS S3 bundle publisher                                                                                                                                                                                                                                                                                                                                   | `false`                                                                                        |
+| `bundlePublisher.awsRolesAnywhereTrustAnchor.region`                                         | AWS region to store the trust bundle                                                                                                                                                                                                                                                                                                                                 | `""`                                                                                           |
+| `bundlePublisher.awsRolesAnywhereTrustAnchor.trustAnchorID`                                  | AWS trust anchor ID to publish to                                                                                                                                                                                                                                                                                                                                    | `""`                                                                                           |
+| `bundlePublisher.awsS3.enabled`                                                              | Enable the AWS S3 bundle publisher                                                                                                                                                                                                                                                                                                                                   | `false`                                                                                        |
+| `bundlePublisher.awsS3.region`                                                               | AWS region to store the trust bundle                                                                                                                                                                                                                                                                                                                                 | `""`                                                                                           |
```

Comment:
We usually have the aws_region as an env var. Do you know if its possible to pull that in here?

Reply:
I don't. I only added the settings documented in the plugin settings.